docs/weave-tutorial/04-models.md

```diff
@@ -0,0 +1,131 @@
+---
+id: models
+title: Models
+sidebar_label: Models
+---
+
+> [PR#6](https://github.com/iov-one/tutorial/pull/6): _Create models_
+
+We defined our state in [codec section](weave-tutorial/03-codec.md). To use models in weave we have to wrap our model with some functionalities and enforce it is a **morm.Model**
+
+Ensure our `OrderBook` fulfills morm.Model. This is just a helper so the compiler will complain loudly here if you forget to implement a method. Guaranteeing it *I am trying to implement this interface*.
+
+```go
+var _ morm.Model = (*OrderBook)(nil)
+```
+
+Now let's explain on our model's identity
+
+## Auto incremented identities
+
+`morm.Model` covers auto-incremented IDs for you. All you have to define `GetID` and `SetID` methods. If you defined `bytes id = 2 [(gogoproto.customname) = "ID"];`  on `codec.proto` you do not even need to write `GetID` method by yourself, Thanks to prototool it will be generated automatically. You will only need to define SetID method.
```

Comment:
```suggestion
`morm.Model` covers auto-incremented IDs for you. All you have to do is define `GetID` and `SetID` methods. If you defined `bytes id = 2 [(gogoproto.customname) = "ID"];`  on `codec.proto` you do not even need to write `GetID` method by yourself, Thanks to prototool it will be generated automatically. You will only need to define SetID method.
```